app/src/main/java/org/thoughtcrime/securesms/ui/theme/ThemeFromPreferences.kt

```diff
@@ -17,38 +17,25 @@ import org.session.libsession.utilities.TextSecurePreferences.Companion.YELLOW_A
  * Some behaviour is hardcoded to cater for legacy usage of people with themes already set
  * But future themes will be picked and set directly from the "Appearance" screen
  */
-@Composable
-fun TextSecurePreferences.getComposeTheme(): ThemeColors {
+val TextSecurePreferences.colors: MaybeFollowSystemColors get() {
```

Comment:
Any reason this is a val and not a function? It feels clearer to me that _something_ is being done when it's called later if this is a function, rather than looking in and seeing that there is a custom getter that has some logic in it.

Reply:
changed back